opencensus/log/__init__.py

```diff
@@ -14,12 +14,13 @@
 
 from collections import namedtuple
 from copy import copy
-import logging
+from logging import (getLogger, LoggerAdapter,
```

Comment:
This seems to be against http://google.github.io/styleguide/pyguide.html#224-decision.